app/src/main/res/layout/music_search_container.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think this file isn't used

Reply:
Yeah you're right, I must have missed removing this file. Removed it